cxplat/inc/cxplat_memory.h

```diff
@@ -38,7 +38,7 @@ typedef struct _cxplat_utf8_string
     }
 
 /**
- * @brief Allocate memory.
+ * @brief Allocate memory. cxplat_allocate_with_tag() should normally be used instead of this API.
```

Comment:
Can we aim to completely remove this? MsQuic's abstraction doesn't have it, fyi.

Reply:
Done